src/main/resources/spring/camel-context.xml

```diff
@@ -1,113 +1,121 @@
-        </filter>
-      </idempotentConsumer>
-      <onException id="_onException2">
-        <exception>java.lang.Exception</exception>
-        <log id="fnf_log" message="EXCEPTION \n ${exception.stacktrace}"/>
-      </onException>
-    </route>
-    <route id="nndssQueue">
-      <from id="foodNetFrom" uri="direct:nndssQueue"/>
-      <idempotentConsumer id="_idempotentConsumer1" messageIdRepositoryRef="nndssIDRepo">
-        <header>CBR_ID</header>
-        <to id="_to4" uri="sdpqueue:{{nndss.queue}}"/>
-      </idempotentConsumer>
-      <onException id="_onException3">
-        <exception>java.lang.Exception</exception>
-        <log id="nndss_log" message="EXCEPTION \n ${exception.stacktrace}"/>
-      </onException>
-    </route>
-    <route id="FoodNetRoute">
-      <from id="_from3" uri="sdpqueue:{{foodNet.queue}}"/>
-      <setHeader headerName="CamelAwsS3Key" id="setAwsS3Key">
-        <simple trim="false"> ${in.header.CBR_ID} 
+    xmlns:camel="http://camel.apache.org/schema/spring"
+    xmlns:context="http://www.springframework.org/schema/context"
+    xmlns:jee="http://www.springframework.org/schema/jee"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.springframework.org/schema/jee http://www.springframework.org/schema/jee/spring-jee.xsd http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd                                http://camel.apache.org/schema/spring http://camel.apache.org/schema/spring/camel-spring.xsd                        ">
+    <!-- START SNIPPET: e1 -->
+    <bean class="org.apache.commons.dbcp.BasicDataSource"
+        destroy-method="close" id="phinMsDataSource">
+        <property name="driverClassName" value="${phinms.jdbc.driverClassName}"/>
+        <property name="url" value="${phinms.jdbc.url}"/>
+        <property name="username" value="${phinms.jdbc.username}"/>
+        <property name="password" value="${phinms.jdbc.password}"/>
+    </bean>
+    <bean class="org.apache.commons.dbcp.BasicDataSource"
+        destroy-method="close" id="sdpqDataSource" primary="true">
+        <property name="driverClassName" value="${sdpqDataSource.jdbc.driverClassName}"/>
+        <property name="url" value="${sdpqDataSource.jdbc.url}"/>
+        <property name="username" value="${sdpqDataSource.jdbc.username}"/>
+        <property name="password" value="${sdpqDataSource.jdbc.password}"/>
+    </bean>
+    <bean class="org.cdc.gov.sdp.PhinMSTransformer" id="myProcessor"/>
+    <bean class="org.cdc.gov.sdp.aphl.AIMSHeaderProcessor" id="aimsHeaderProcessor"/>
+    <bean class="org.cdc.gov.sdp.JSONTransformer" id="jsonTransformer"/>
+    <bean class="org.cdc.gov.sdp.ArrayListAggregationStrategy" id="agg"/>
+    <bean class="org.cdc.gov.sdp.HTTP4Transformer" id="httpTransformer"/>
+    <bean class="org.cdc.gov.sdp.SDPMessageIdRepository" id="foodNetIDRepo">
+    	<constructor-arg ref="nndssDataSource"/>
```

Comment:
I think this is probably copied over from tests.  There is no nndssDataSource in this route, can you change it to sdpqDataSource